tools/WinObjC.Tools/vsimporter-templates/project.json

```diff
@@ -0,0 +1,20 @@
+{
+  "dependencies": {
+    "WinObjC.Frameworks": "*",
```

Comment:
do we want to just go ahead and give them packaging anyway? Unless they have a .nuproj its harmless and would make it easier for them to add after the fact?

---
In reply to: [100940735](https://github.com/Microsoft/WinObjC/pull/1958#discussion_r100940735) [](ancestors = 100940735)